### PR TITLE
Updated copilot handoff successCardHandoff and lab instructions for m…

### PR DESCRIPTION
**Pull request by Carter425**
- …ore handoff details

An Overview of what the changes are
1. Updated README to link to new Copilot Handoff Code Walkthrough section
2. Updated set-up to link to the correct repository where the copilot handoff sample currently is located. (Someone should move this sample to https://github.com/OfficeDev/Copilot-for-M365-Plugins-Samples.git, but it is not currently located there so we should have the correct repository linked)
3. Updated Lab exercise 5 to include information on copilot handoffs
4.  Updated the successCardHandoff adaptive card for match the design what we are showcasing on Learn site
5. Removed the + "-continuation" from the continuation token for new design and as the handoff is highlighted as a new color in the adaptive card
6. Updated image of the successful handoff to match the new adaptive card

### CODE REVIEW

**Review by premkaretimsft — APPROVED** [review]

**Comment by Carter425** [comment]
@premkaretimsft and @Wajeed-msft, I updated the pull request to use the latest agent terminology per copilot wave 2

**Review by premkaretimsft — APPROVED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Comment by Carter425** [comment]
Hi @premkaretimsft, Thanks for approving the changes yesterday. I'm not sure why but it only showing @Wajeed-msft as the only official approved required reviewer here. I'm not sure if I need Prem to reapprove the review or if I need to get another reviewer with write access here.

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Pull request merged**
